src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -157,17 +157,24 @@ LedgerEntryHandler::process(LedgerEntryHandler::Input input, Context const& ctx)
         return Error{*status};
 
     auto const lgrInfo = std::get<ripple::LedgerHeader>(lgrInfoOrStatus);
-    auto const ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
-
-    if (!ledgerObject || ledgerObject->empty())
-        return Error{Status{"entryNotFound"}};
+    auto output = LedgerEntryHandler::Output{};
+    output.deletedLedgerIndex = std::nullopt;
+    auto ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
+
+    if (!ledgerObject || ledgerObject->empty()) {
+        if (not input.includeDeleted) 
+            return Error{Status{"entryNotFound"}};
+        ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq - 1, ctx.yield);
+        if (!ledgerObject || ledgerObject->empty()) 
```

Comment:
If an object is deleted at seq, it must exist for seq-1, right? We can use an ASSERT here.